src/sec_certs/dataset/cc.py

```diff
@@ -256,12 +233,6 @@ def from_web_latest(cls) -> CCDataset:
     def _set_local_paths(self):
         super()._set_local_paths()
 
-        if self.auxiliary_datasets.pp_dset:
-            self.auxiliary_datasets.pp_dset.json_path = self.pp_dataset_path
-
-        if self.auxiliary_datasets.mu_dset:
-            self.auxiliary_datasets.mu_dset.root_dir = self.mu_dataset_dir
-
```

Comment:
Is this set properly in the new API?

Reply:
Yes, see https://github.com/crocs-muni/sec-certs/blob/e6cab194fad1b924b060cf91f4b39e13b1cd55a1/src/sec_certs/dataset/dataset.py#L238-L240